frontend/app/utils/application-code-utils.ts

```diff
@@ -21,3 +55,12 @@ export function formatSubmissionApplicationCode(applicationCode: string): string
   if (!/^\d{13}$/.test(strippedCode)) return applicationCode;
   return (strippedCode.match(/....$|.../g) ?? []).join(' ');
 }
+
+/**
+ *
+ * @param code - the application code
+ * @returns a formatted code using a supplied separator
+ */
+export function extractDigits(input: string) {
+  return input.replace(/\D/g, '');
+}
```

Comment:
Might be a nitpick but I feel `extractDigits(..)` belongs in a more general utils file like `string-utils` rather than `application-code-utils`. 

The documentation for this function should be adjusted since the `input` doesn't necessarily have to be an application code.